network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html

```diff
@@ -1,6 +1,12 @@
 {% extends "./mini_site_name_space.html" %}
 {% load wagtailcore_tags mini_site_tags %}
 
+Note: we don't want the donation modal to pop up on
```

Comment:
should this be in a comment block?

Reply:
As an `extends`, nothing outside of `{%block%}` codes is active code, so it doesn't have to be marked as comment. That said, trivial to add it in so why not.